src/box/txn.c

```diff
@@ -1103,6 +1104,11 @@ txn_commit_impl(struct txn *txn, enum txn_commit_wait_mode wait_mode)
 				 "Non-blocking commit of a synchronous txn");
 			goto rollback_abort;
 		}
+		if (wait_mode == TXN_COMMIT_WAIT_MODE_NONE &&
+		    journal_queue_is_full()) {
+			diag_set(ClientError, ER_JOURNAL_FULL);
```

Comment:
Users don't know such concept as `JOURNAL`. There's not a single mention of it in documentation. Can we rename it to smth like `ER_WAL_QUEUE_FULL`?

Reply:
Absolutely right, good that you noticed, I changed the error name and message.